src/omg-cli/omg/tiles/healthwatch/configuration.go

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package healthwatch
+
+import (
+	"encoding/json"
+	"fmt"
+	"omg-cli/config"
+	"omg-cli/omg/tiles"
+	"omg-cli/ops_manager"
+)
+
+const (
+	skipSSLValidation = "true"
+)
+
+type Properties struct {
+	OpsManagerURL     tiles.Value `json:".healthwatch-forwarder.opsmanager_url"`
+	BoshHealthCheckAZ tiles.Value `json:".healthwatch-forwarder.health_check_az"`
+	// Endpoint          tiles.Value `json:".properties.firehose_endpoint"`
+	// SkipSSLValidation tiles.Value `json:".properties.firehose_skip_ssl"`
+	// ServiceAccount    tiles.Value `json:".properties.service_account"`
+	// ProjectID         tiles.Value `json:".properties.project_id"`
+}
+
+type Resources struct {
+	// HealthWatchForwarder tiles.Resource `json:"healthwatch-forwarder"`
+}
+
+func (t *Tile) Configure(envConfig *config.EnvConfig, cfg *config.Config, om *ops_manager.Sdk) error {
+	if err := om.StageProduct(tile.Product); err != nil {
+		return err
+	}
+
+	network := tiles.NetworkODBConfig(cfg.ServicesSubnetName, cfg, cfg.DynamicServicesSubnetName)
+
+	networkBytes, err := json.Marshal(&network)
+	if err != nil {
+		return err
+	}
+
+	properties := &Properties{
+		OpsManagerURL:     tiles.Value{fmt.Sprintf("https://opsman.%s", cfg.DnsSuffix)},
+		BoshHealthCheckAZ: tiles.Value{cfg.Zone1},
+		// Endpoint:          tiles.Value{fmt.Sprintf("https://api.sys.%s", cfg.DnsSuffix)},
```

Comment:
are these needed?

Reply:
Yes they are required field of healthwatch

Reply:
thanks! so awesome health watch is apart of this!